packages/compass-generative-ai/src/store/util.ts

```diff
@@ -0,0 +1,8 @@
+import type { AnyAction } from 'redux';
+
+export function isAction<A extends AnyAction>(
```

Comment:
nit, fine as is too, we typically put utils in a separate folder, like:
https://github.com/mongodb-js/compass/blob/main/packages/compass-indexes/src/utils/is-action.ts